elpis/endpoints/pron_dict.py

```diff
@@ -0,0 +1,83 @@
+from flask import request
+from ..blueprint import Blueprint
+from flask import current_app as app, jsonify
+from elpis.wrappers.objects.interface import KaldiInterface
+from elpis.wrappers.objects.pron_dict import PronDict
+from elpis.wrappers.objects.dataset import Dataset
+
+
+bp = Blueprint("pron_dict", __name__, url_prefix="/pron-dict")
+
+
+@bp.route("/new", methods=['GET', 'POST'])
+def new():
+    kaldi: KaldiInterface = app.config['INTERFACE']
+    pd = kaldi.new_pron_dict(request.json['name'])
```

Comment:
It feels very nitpicky, but `pd` and `ds` need more descriptive names.

Reply:
this was based on existing names, eg `m: Model` and `ds: Dataset`. Should expand to the more verbose `model`, `pron_dict`, `dataset` ?